plugins/win-dshow/win-dshow.cpp

```diff
@@ -1746,13 +1758,16 @@ static bool DeviceIntervalChanged(obs_properties_t *props, obs_property_t *p, ob
 			}
 		}
 
+		PRAGMA_WARN_PUSH
+		PRAGMA_WARN_DEPRECATION
 		if (listed_val != val) {
 			obs_data_set_autoselect_int(settings, FRAME_INTERVAL, listed_val);
 			val = listed_val;
 		}
 
 	} else {
 		obs_data_unset_autoselect_value(settings, FRAME_INTERVAL);
+		PRAGMA_WARN_POP
```

Comment:
This is a little unsightly. I might prefer just treating them separately, or wrapping the whole if/else. @PatTheMav ?